public/components/wz-menu/wz-menu.js

```diff
@@ -632,7 +632,7 @@ export const WzMenu = withWindowSize(class WzMenu extends Component {
   }
 
   getApiSelectorComponent() {
-    let style = { maxWidth: 100 };
+    let style = { minWidth: 'max-content' };
```

Comment:
increasing the size the component can take is a good first step, but not limiting it in any way can lead to other issues, if the name is too long still, the other components in the menu will be squished and rendered incorrectly
![image](https://user-images.githubusercontent.com/80431234/124894256-c45a0d80-dfdb-11eb-80bc-80e4eee52116.png)
I suggest you specify a max size and `text-overflow: ellipsis`

Reply:
It is a very difficult case to happen (that the api name is so long) but I will consider it.